GsaGH/Helpers/StringExtension.cs

```diff
@@ -1,6 +1,13 @@
-﻿using System.Linq;
+﻿using System;
+using System.Linq;
 using System.Text.RegularExpressions;
 
+using Grasshopper.Kernel;
+
+using GsaGH.Parameters;
+
+using Rhino.Render.Fields;
```

Comment:
why new namespaces if there is no change?

Reply:
@DominikaLos Good point. Have removed.